README.md

```diff
@@ -12,11 +12,20 @@ Copyright © 2017 FIXME
 
 Distributed under the Eclipse Public License either version 1.0 or (at
 your option) any later version.
-
-
-## NOTES
-- you need to provide a secrets.clj file with
+  
+## Getting Started
+This project uses re-natal which in turn wraps react-native (v0.41.1). You would need to follow their installation instruction: 
+- [re-natal](https://github.com/drapanjanas/re-natal)
+- [react-native](https://facebook.github.io/react-native/docs/getting-started.html)
+
```

Comment:
@mehdisadeghi this should be fixed since the gradle wrapper is already part of `master`. Please see my comment in the issue that you openned ;)